README.md

```diff
@@ -22,3 +22,35 @@ somewhere in your path). You can also install (by default directly to `/usr`)
 via `make install`.
 
 On FreeBSD (and, I expect, other non-GNU platforms), you will want to use gmake.
+
+
+FAQ
+---
+
+Q: What platforms are supportded?  
+A: keynav should work on nearly any Unix-like that runs X11. It has been
+confirmed to work on extremely varied GNU/Linux systems (incuding RPM-based,
+Debian derivatives, musl-based systems, and Arch), and FreeBSD. If you get it to
+run elsewhere, please let me know so I can add it to the list. If you try to run
+it on another Unix-like and have trouble, please get in touch and I'll try to
+help. If attempting to run elsewhere, note that we currently have a dependency
+on GNU Make (gmake), and it hasn't been tested with many compilers yet.
+
+Q: Does it work on Android/Windows/Wayland/iOS/...?  
+A: Sadly, no; keynav is totally dependent on X11, and porting it to any other
+graphical system would really be a clone/rewrite. Although I am aware of no
+exact analogues on other systems, I suggest looking into Tasker (Android),
+AutoHotKey (Windows), and AppleScript (macOS). If you find something that works,
+let me know and I'll consider adding it to this list.
+
+Q: Can I use keynav to scroll?  
```

Comment:
I'm sorry if this information is easy to find, but is there a way to do control-click from keynav?
I tried putting "ctrl+1 click 1" in my .keynavrc but it had no effect - no click happens when i ctrl+1
(i also tried with ctrl-q and there's still no effect)
I just want to be able to control-click on something...am i missing something?  thank you